routes.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func getCentres(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	data, err := db.Centres.GetAll()
```

Comment:
Handle the error before calling the next function that might return another error.